src/main/java/graphql/schema/idl/SchemaGeneratorHelper.java

```diff
@@ -292,8 +293,13 @@ Object buildArrayValue(GraphQLType requiredType, ArrayValue arrayValue) {
 
     Object buildObjectValue(ObjectValue defaultValue, GraphQLInputObjectType objectType) {
         Map<String, Object> map = new LinkedHashMap<>();
-        defaultValue.getObjectFields().forEach(of -> map.put(of.getName(),
-                buildValue(of.getValue(), objectType.getField(of.getName()).getType())));
+        objectType.getFieldDefinitions().forEach(
+                f -> {
+                    final Value<?> fieldValueFromDefaultObjectValue = defaultValue.getObjectFields().stream()
```

Comment:
Performance of this statement can be improved by keeping a map of Object field's name and the field itself in ObjectValue class.

Reply:
ok, thanks